core/base/ftmTree/FTMTreeUtils_Template.h

```diff
@@ -82,6 +83,24 @@ namespace ttk {
     // --------------------
     // Get
     // --------------------
+    template <class dataType>
+    int FTMTree_MT::getMergedRootOrigin() {
```

Comment:
ttk has simplex types ?

Reply:
In the next commit I replaced the type returned by this function by idNode (instead of int).
(But I'm not sure I correctly understood the question)